db/migrate/20250722121803_rename_pwd_auth_session_token_to_reporting_app_session_token.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RenamePwdAuthSessionTokenToReportingAppSessionToken < ActiveRecord::Migration[
+  8.0
+]
+  def change
+    rename_column :users, :pwd_auth_session_token, :reporting_app_session_token
```

Comment:
I think we could probably combine this migration with the previous one to avoid needing to create the column and then rename it.